### ISSUE TITLE
Required fields are not announced by screen reader

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-accessibility-passport#111]
From AAACT:

> The required fields are not announced by the screen reader.
> The “required” attribute could be set on the input fields.
> Alternative advisory technique if no HTML validation will be performed: [ARIA2: Identifying a required field with the aria-required property](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA2.html)

> Although validations are not ready to fully test against this criterion, the current situation is that the error messages are not announced by the screen reader.

### Screenshot
<img width="377" alt="image" src="https://user-images.githubusercontent.com/3046459/170296460-0521d606-fffe-4710-b8e8-0956c01ff8a8.png">
